edx_proctoring/templates/onboarding_exam/submitted.html

```diff
@@ -45,6 +45,13 @@
       </button>
     </p>
   {% endif %}
+  <p>
```

Comment:
I can't comment on the line, but line 15 also references identity verification with respect to onboarding. I think we should also remove it there.

If you choose to do that, there's also an errant comma.

```
 If you do not have an onboarding profile with the system, then Verificient will review your submission and create an onboarding profile to grant you access to proctored exams.  Onboarding profile review can take 2+ business days.
```